src/components/configuration/DataLakeAlertsConfig.vue

```diff
@@ -0,0 +1,622 @@
+    )
+  )
+})
+
+const alertLevelOptions = [
+  { title: 'Success', value: AlertLevel.Success },
+  { title: 'Info', value: AlertLevel.Info },
+  { title: 'Warning', value: AlertLevel.Warning },
+  { title: 'Error', value: AlertLevel.Error },
+  { title: 'Critical', value: AlertLevel.Critical },
+]
+
+// Methods
+const loadAlerts = (): void => {
+  alerts.value = getAllDataLakeAlerts()
+}
+
+const loadVariables = (): void => {
+  availableVariablesMap.value = getAllDataLakeVariablesInfo()
+}
+
+const getVariableName = (variableId: string): string => {
+  const variable = getDataLakeVariableInfo(variableId)
+  return variable?.name || variableId
+}
+
+/**
+ * Copy the variable name to clipboard for use in custom messages
+ * @param {string} variableId - The ID of the variable to copy the name for
+ */
+const copyVariableName = async (variableId: string): Promise<void> => {
+  if (!variableId) return
+  const variableName = getVariableName(variableId)
+  const placeholder = `{{${variableName}}}`
+  try {
+    await navigator.clipboard.writeText(placeholder)
+  } catch (error) {
+    console.error('Failed to copy to clipboard:', error)
+  }
+}
+
+/**
+ * Get a summary of all variables used in an alert's conditions
+ * @param {DataLakeAlert} alert - The alert to get the variables summary for
+ * @returns {string} The summary of the variables used in the alert's conditions
+ */
+const getAlertVariablesSummary = (alert: DataLakeAlert): string => {
+  const uniqueVariableIds = [...new Set(alert.conditions.map((c) => c.variableId))]
+  if (uniqueVariableIds.length === 0) return 'No variables'
+  const names = uniqueVariableIds.map((id) => getVariableName(id))
+  return names.join(', ')
+}
+
+const getLevelClass = (level: AlertLevel): string => {
+  const classes: Record<AlertLevel, string> = {
+    [AlertLevel.Success]: 'bg-green-600',
+    [AlertLevel.Info]: 'bg-blue-600',
+    [AlertLevel.Warning]: 'bg-yellow-600',
+    [AlertLevel.Error]: 'bg-red-600',
+    [AlertLevel.Critical]: 'bg-red-800',
+  }
+  return classes[level] || 'bg-slate-600'
+}
+
+const toggleAlertEnabled = (alert: DataLakeAlert): void => {
+  setDataLakeAlertEnabled(alert.id, !alert.enabled)
+  loadAlerts()
+}
+
+/**
+ * Handle when a condition's variable is changed - reset operator and value to defaults
+ * @param {AlertCondition} condition - The condition to handle the variable change for
+ */
+const onConditionVariableChanged = (condition: AlertCondition): void => {
+  const type = getConditionVariableType(condition)
+  if (type) {
+    const operators = getComparisonOperatorsForType(type)
+    condition.operator = operators[0]?.value || 'equals'
+    // Set default compare value based on type
+    switch (type) {
+      case 'boolean':
+        condition.compareValue = true
+        break
+      case 'number':
+        condition.compareValue = 0
+        break
+      default:
+        condition.compareValue = ''
+    }
```

Comment:
Should this be using the `getDefaultCompareValue` function from the typescript file?